releasenotes/notes/37946.yaml

```diff
@@ -3,4 +3,4 @@ kind: bug-fix
 area: istioctl
 releaseNotes:
   - |
-    **Fixed** server-side apply detection logic to not run for k8s cluster version < 1.22
+    **Fixed** Server Side Apply is enabled by default for Kubernetes cluster versions above 1.22
```

Comment:
```suggestion
    **Fixed** Server Side Apply to not run for Kubernetes cluster versions below 1.22
```

Reply:
Hi @dhawton 
After John's initial review, the logic was changed to:

```
if v>22
  ssa is enabled
else if v>18
  ssa detection logic is run
```

Reply:
so SSA will be used for versions below 22 if it is detected (same behaviour as earlier)

Reply:
Fixed my suggestion based on that logic.